com.dreamsourcelab.DSView.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.dreamsourcelab.DSView
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: DSView
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - '*.a'
+  - '*.la'
+  - /man
+  - /share/info
+rename-icon: dsview
+rename-desktop-file: dsview.desktop
+modules:
+  - name: dsview
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=RelWithDebInfo'
+    sources:
+      - type: git
+        url: https://github.com/DreamSourceLab/DSView.git
+        tag: v1.3.0
+        commit: 02152f1c75f316b35558b494dbf22f9fb12a92bb
+      - type: patch
+        path: patches/fix_cmake_install_prefix.patch
+      - type: file
+        path: com.dreamsourcelab.DSView.metainfo.xml
+    build-commands:
+      - install -Dm644 com.dreamsourcelab.DSView.metainfo.xml -t "${FLATPAK_DEST}/share/metainfo"
+    modules:
+      - shared-modules/libusb/libusb.json
+      - name: boost
+        sources:
+          - type: archive
+            url: https://boostorg.jfrog.io/artifactory/main/release/1.83.0/source/boost_1_83_0.tar.gz
+            sha256: c0685b68dd44cc46574cce86c4e17c0f611b15e195be9848dfd0769a0a207628
+        buildsystem: simple
+        build-commands:
+          - ./bootstrap.sh --prefix=${FLATPAK_DEST}
+          - ./b2 install
+      - name: fftw
+        sources:
+          - type: archive
+            url: https://www.fftw.org/fftw-3.3.10.tar.gz
+            sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+        buildsystem: autotools
+      - name: fake_udev_dir     # workaround the udev rule installation procedure in CMakeLists.txt
+        buildsystem: simple
+        build-commands:
+          - mkdir -p ${FLATPAK_DEST}/lib/udev/rules.d
```

Comment:
do you really need this?

Reply:
It is certainly a no-op for flatpak, but I need to find a way to make CMake happy so, should I:

- Completely bypassing udev rules installation in CMakeLists with patch, thus needs no fake dir
- Leave it as-is, to satisfy the upstream installing behavior (patch still required but may fly upstream)